feeds/wifi-ax/hostapd/patches/601-fix_ubus_assoc.patch

```diff
@@ -0,0 +1,26 @@
+--- a/src/ap/ieee802_11.c
```

Comment:
I believe we should apply the patch on top of the latest patch. Sometimes the hunk succeeds but it might corrupt the other patches.